bosh-tooling-uaa.yml

```diff
@@ -39,7 +39,7 @@ instance_groups:
             <<: *client-template
           # For authentication to bosh via concourse
           concourse_bosh:
-            secret: (( param "specify concourse bosh client secret" ))
+            secret: (( grab secrets.bosh_uaa_concourse_bosh_client_secret ))
```

Comment:
isn't this supposed to be `secrets.bosh_uaa_ci_client_secret` or am I missing something?

Reply:
concourse uses a different client for interacting with tooling bosh https://github.com/18F/cg-deploy-bosh/pull/179/files#diff-688ee30f2b0e35b63848f09d2c9bbbb3R669 AFACT we should probably update to use `ci` like other deployments